app/main/appium.js

```diff
@@ -301,6 +301,7 @@ function connectClientMethodListener () {
       fetchArray = false, // Optional. Are we fetching an array of elements or just one?
       elementId, // Optional. Element being operated on 
       args = [], // Optional. Arguments passed to method
+      skipScreenshot = false, // Optional. Do we want the updated source and screenshot?
```

Comment:
perhaps, we can select better name for this variable if it also affects source update?

Reply:
Good call